Packs/FeedProofPoint/Integrations/FeedProofPoint/FeedProofPoint.yml

```diff
@@ -0,0 +1,122 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: ProofpointFeed
+  version: -1
+configuration:
+- display: Fetch indicators
+  name: feed
+  required: false
+  type: 8
+- defaultvalue: '240'
```

Comment:
Please update the default values of your parameters using this [doc](https://demisto.slack.com/archives/CRT0G4Y82/p1579794243004000). Also, update the `description.md` file to include the descriptions of the subfeeds that are in that document as well.